src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -2142,14 +2273,17 @@ bool isRefEscape
         }
 
         /// <summary>
-        /// Returns the set of arguments to be considered for escape analysis of a method invocation.
-        /// Each argument is returned with the corresponding parameter and ref kind. Arguments are not
-        /// filtered - all arguments are included exactly once in the array, and the caller is responsible for
-        /// determining which arguments affect escape analysis. This method is used for method invocation
-        /// analysis, regardless of whether UseUpdatedEscapeRules is set.
+        /// Returns the set of arguments to be considered for escape analysis of a method invocation. This
+        /// set potentially includes the receiver of the method call.  Each argument is returned (only once)
+        /// with the corresponding parameter and ref kind.
+        /// 
+        /// No filtering like removing non-reflike types is done by this method. It is the responsibility of
+        /// the caller to determine which arguments impact escape analysis.
+        ///
+        /// This method is used by for old and new escape rules to collect information.
```

Comment:
> This method is used by for old and new escape rules to collect information.

I'm not sure what this sentence is trying to say

Reply:
A lot of the escape methods are specialized for the old and updated rules. This was trying to say it is one of the few methods that is used by both to collect information. Tried to clarify a bit but couldn't find good language. Ended up removing this sentence.